packages/grafbase-sdk/src/graph.ts

```diff
@@ -0,0 +1,9 @@
+import { GrafbaseFederationSchema, GrafbaseSchema } from './grafbase-schema'
+
+/**
+ * A builder for a Grafbase schema definition.
+ */
+export const graph = {
+  Federation: () => new GrafbaseFederationSchema(),
+  Single: () => new GrafbaseSchema()
```

Comment:
I would use `SingleGraph()` and `FederatedGraph()`

Reply:
See above, we do we want to do this now or in another PR?

Reply:
Wait, do you mean for the external interface? I went with the Linear issue but we can change the names, sure

Reply:
@fbjork Are these free functions or under `graph`? `graph.SingleGraph()` is a bit redundant

Reply:
Or do you mean for the internal classes?

Reply:
for the internal object. `new SingleGraph()`

Reply:
I see, ok will do

Reply:
Done